engine/classes/ElggSession.php

```diff
@@ -250,11 +254,67 @@ public function isLoggedIn() {
 	 * @since 1.9
 	 */
 	public function removeLoggedInUser() {
-		$this->loggedInUser = null;
+		$this->logged_in_user = null;
 		$this->remove('guid');
 	}
 
 	/**
+	 * Get current ignore access setting.
+	 *
+	 * @return bool
+	 */
+	public function getIgnoreAccess() {
+		return $this->ignore_access;
+	}
+
+	/**
+	 * Set ignore access.
+	 *
+	 * @param bool $ignore Ignore access
+	 *
+	 * @return bool Previous setting
+	 */
+	public function setIgnoreAccess($ignore = true) {
+		_elgg_services()->accessCache->clear();
+
+		$prev = $this->ignore_access;
+		$this->ignore_access = $ignore;
+
+		return $prev;
+	}
+
+	// @codingStandardsIgnoreStart
+	/**
+	 * Alias of getIgnoreAccess()
+	 *
+	 * @todo remove with elgg_get_access_object()
+	 *
+	 * @return bool
+	 * @deprecated 1.8 Use elgg_get_ignore_access()
+	 */
+	public function get_ignore_access() {
+		return $this->getIgnoreAccess();
```

Comment:
These don't throw deprecation notices because they would only be accessed via elgg_get_access_object(), which does.